### PR TITLE
add: added homepage

### PR DESCRIPTION
**Pull request by devsurajrai**
Added homepage to the e-commerce project. Please Review the PR. Things to remember while reviewing:
- Folder Structure
- Coding Style
- Components Breakdown

Screenshot 2022-03-17 at 6 20 20 PM

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+import { Header } from "./components/Header.component/Header";
+import Home from "./pages/Home";
 import Login from "./pages/Login";
 import { SignUp } from "./pages/SignUp";
 
 const App = () => (
-  <>
+  <div className="e-commerce-app-container">
     <Header />
     {/* <Login /> */}
```
Comment (vaishnavipaliya11) [inline src/App.js:10]: Please remove the commented code

File: src/App.js [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
-    <SignUp />
-  </>
+    {/* <SignUp /> */}
```
Comment (vaishnavipaliya11) [inline src/App.js:11]: Please remove the commented code

**Review by vaishnavipaliya11 — CHANGES_REQUESTED** [review]
Please remove commented code. Rest all looks good.

**Review by saifudinbadani — APPROVED** [review]
Maybe you can put all the css of Homepage components in one file and all files of homepage components in same folder, it will be helpful when you scale the app.

**Comment by saifudinbadani** [comment]
Otherwise, the code looks clean.

**Pull request merged**